backend/src/main/resources/db/migration/V0.34__create_tables_for_upload_processing_status.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE verkuendung_import_processing_status AS ENUM ('CREATED', 'PROCESSING', 'SUCCESS', 'ERROR');
+
+CREATE TABLE IF NOT EXISTS verkuendung_import_processes (
+    id UUID NOT NULL PRIMARY KEY,
+    status verkuendung_import_processing_status NOT NULL DEFAULT 'CREATED',
+    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    started_at TIMESTAMPTZ,
+    finished_at TIMESTAMPTZ
+);
+
+CREATE TABLE IF NOT EXISTS verkuendung_import_process_detail (
+    id UUID NOT NULL PRIMARY KEY,
+    process_id UUID REFERENCES verkuendung_import_processes(id),
```

Comment:
We could also think about creating an index on the foreign key because even if we won't be doing JOINS (since with lazy loading hibernate will do a separate SELECT on demand) the filtering using the process_id will benefit from the index.

```
CREATE INDEX idx_verkuendung_process_detail_process_id 
ON verkuendung_import_process_detail(process_id);
```